pipelines/base/peribolos-run.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: peribolos-run
+spec:
+  params:
+  - name: REPO_NAME
+    type: string
+    default: '.github'
+  - name: CLIENT_ID
```

Comment:
This should be **Installation ID**, client ID is a different thing - unique per application, shared for all installations.

![image](https://user-images.githubusercontent.com/7453394/161759171-a62b992a-4ddc-4299-b589-984f4d0e7594.png)

Reply:
Fixed, forgot there is a difference :smile:.